blog/content/post/2020-08-05-PCIe-features.md

```diff
@@ -0,0 +1,213 @@
+---
+title: What features PCIe has?
+abstract: 'Introductory blog post to PCIe features.
+          In this article you can read what PCIe capability is
+          and see examples of such capabilities.'
+cover: /covers/PCIExpress.jpg
+author: marek.kasiewicz
+layout: post
+published: true
+date: 2020-08-05
+archives: "2020"
+
+tags:
+  - Firmware
+categories:
+  - Firmware
+
+---
+
+## What this blog is about
+
+Goal of this post is to introduce readers to the concept of PCI Express
+capabilities. It briefly describes what capability is and how it works. It also
+lists some example capability described in PCI Express (PCIe) specification.
+
+## Introduction
+
+PCI Express uses inverted tree topology. It starts from the Root Complex which
```

Comment:
I miss some one introductory sentence here... I know that you are an engineer and not a writer, but try to add just some opening before starting with your topic

Reply:
Yes, when I read this post now I can see how far it is from a post written by a good writer ^^
As for the introduction, what would you say about adding something like this:
"
Probably everyone knows about the existence of the PCIe standard. It is commonly
used in PCs to connect various devices to the motherboard. Every possible modern
graphic card uses PCIe, most sound cards use it, it starts to become the standard
even for SSDs, and that's just a tip of the iceberg. But how does the OS communicate
with all of these devices and how does it know what these devices are capable of?
This post will bring you closer to knowing the answer for those questions, but
let's start from the beginning...
"